documents-webapp/src/main/webapp/vue-app/trash-management/components/menu/DocumentsTrashItemMenu.vue

```diff
@@ -51,5 +72,38 @@ export default {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
     }
   },
+  watch: {
+    menu() {
+      // Workaround to fix closing menu when clicking outside
+      if (this.menu) {
+        document.addEventListener('mousedown', this.closeMenu);
+      } else {
+        document.removeEventListener('mousedown', this.closeMenu);
+      }
+    },
+  },
+  methods: {
+    restoreDocument() {
+      this.loading = true;
+      this.$trashManagementService.restoreDocument(this.trashElementItem.path).then((resp) => {
+        if (resp.ok) {
+          this.displayAlert(this.$t('trash.element.restore.message.success'));
+          this.$root.$emit('trash-elements-updated');
+        }
```

Comment:
the function restoreDocument returns always an object when the response is 200, no need for this extra check IMO

Reply:
check removed !